core/src/apps/nem/multisig/layout.py

```diff
@@ -61,7 +61,7 @@ async def _require_confirm_address(action: str, address: str) -> None:
     await confirm_address(
         TR.nem__confirm_address,
         address,
-        action,
-        "confirm_multisig",
-        ButtonRequestType.ConfirmOutput,
+        description=action,
```

Comment:
not sure why you added these keywords everywhere to `confirm_address`

Reply:
Understood! Note that using keyword arguments takes a little tiny bit more flash size, that's why they were removed on many places in the past. See here: https://github.com/grdddj/upysize#12-do-not-use-keyword-arguments 

Don't revert, just something to keep in mind.